migration_guides/from2.2to2.3.md

```diff
@@ -0,0 +1,35 @@
+# Migration Guide (From 2.2 to 2.3)
+
+*This page is updated to the PuMuKIT 2.3.0 version*
+
+Here we describe the process to upgrade from PuMuKIT 2.2 to PuMuKIT 2.3. If you are trying to upgrade from PuMuKIT 1, please check out [this guide](from1.7to2.1) instead and then [this_guide](from2.1to2.2).
+
+1. First let's open the PuMuKIT 2 folder on our terminal.
+
+    ```bash
+    cd /path/to/pumukit
+    ```
+2. Then we checkout the most recent stable-release for the 2.3 version. (In this case the 2.3.0 tag)
+
+    ```bash
+    git checkout 2.3.0
+    ```
+3. We should delete vendor folder and then run composer install (composer should be installed globally, if not check out the [composer docs](https://getcomposer.org/doc/00-intro.md))
+
+    ```bash
+    rm -rf vendor/
+    composer install
```

Comment:
You can give the alternative of using the `composer.phar` file including in the PuMuKIT installation.
```
php composer.phar install
```